Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -167,16 +167,25 @@ class BrowserViewController: UIViewController {
         DefaultBrowserExperiment.minPromoSearches() <= User.shared.searchCount
     }
     fileprivate var shouldShowWhatsNewPageScreen: Bool { whatsNewDataProvider.shouldShowWhatsNewPage }
-    fileprivate var shouldShowAPNConsentScreen: Bool {
-        EngagementServiceExperiment.isEnabled &&
-        EngagementServiceExperiment.minSearches() <= User.shared.searchCount &&
-        User.shared.shouldShowAPNConsentScreen
-    }
 
     let whatsNewDataProvider = WhatsNewLocalDataProvider()
     
     let referrals = Referrals()
     var menuHelper: MainMenuActionHelper?
+    
+    // Ecosia: Initialize the OptInReminderManager that handles the APNConsent showing
+    private static var userApnConsentOptInReminderManager = User.shared.apnConsentReminderModel ?? OptInModel() {
```

Comment:
Since this is the `OptInModel` and not the `OptInReminderManager`, how about naming it `userApnConsentOptInModel`? I feel the current name is confusing with the variable below

Reply:
Totally! Let me rename it then 💪